pkg/ui/ui.go

```diff
@@ -170,6 +170,12 @@ func GetWebPrefix(logger log.Logger, externalPrefix, prefixHeader string, r *htt
 		level.Warn(logger).Log("msg", "Could not parse value of UI external prefix", "prefix", prefix, "err", err)
 	}
 
+	// To use relative URLs we need the prefix to have trailing "/" so that we don't have to add the
+	// "/" everywhere in templates.
+	if prefix != "" {
+		prefix = prefix + "/"
```

Comment:
What if a prefix is given that already ends in `/`? Can we ensure we only have one trailing slash?

Reply:
That is already being done in the sanitizePrefix function.